<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>registroWin</class>
 <widget class="QWidget" name="registroWin">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>950</width>
    <height>600</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>950</width>
    <height>600</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>950</width>
    <height>600</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QLabel" name="bg">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>961</width>
     <height>601</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="pixmap">
    <pixmap resource="resource.qrc">:/resource/img/registro.png</pixmap>
   </property>
  </widget>
  <widget class="QStackedWidget" name="sw">
   <property name="geometry">
    <rect>
     <x>120</x>
     <y>70</y>
     <width>711</width>
     <height>461</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">
background: rgba(0,0,0,0);</string>
   </property>
   <property name="currentIndex">
    <number>1</number>
   </property>
   <widget class="QWidget" name="cand">
    <widget class="QLabel" name="label">
     <property name="geometry">
      <rect>
       <x>210</x>
       <y>10</y>
       <width>291</width>
       <height>41</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Roboto</family>
       <pointsize>25</pointsize>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">color: white;</string>
     </property>
     <property name="text">
      <string>Registro de Candidatos</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
    <widget class="QPushButton" name="btnRegistrarCand">
     <property name="geometry">
      <rect>
       <x>370</x>
       <y>410</y>
       <width>171</width>
       <height>31</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>-1</pointsize>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">border: 0px solid;
background: #EF40A7;
color:white;
border-radius: 15px;
font-weight:bold;
font-size:15px;
</string>
     </property>
     <property name="text">
      <string>REGISTRARSE</string>
     </property>
    </widget>
    <widget class="QPushButton" name="btnCandidato">
     <property name="geometry">
      <rect>
       <x>160</x>
       <y>410</y>
       <width>171</width>
       <height>31</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>-1</pointsize>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">border: 2px solid  #EF40A7;
background: rgba(0,0,0,0);
color:white;
border-radius: 15px;
font-size:15px;
</string>
     </property>
     <property name="text">
      <string>Soy Votante</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="inNombreCand">
     <property name="geometry">
      <rect>
       <x>50</x>
       <y>80</y>
       <width>271</width>
       <height>31</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>12</pointsize>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">color: grey;
border: solid 0px white;
background: white;
border-radius:10px;
padding:8px;</string>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QComboBox" name="inEstadoCand">
     <property name="geometry">
      <rect>
       <x>50</x>
       <y>150</y>
       <width>271</width>
       <height>32</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">color: grey;
border: solid 0px white;
background: white;
border-radius:10px;
padding:8px;</string>
     </property>
    </widget>
    <widget class="QLabel" name="lblUsuario_4">
     <property name="geometry">
      <rect>
       <x>50</x>
       <y>50</y>
       <width>101</width>
       <height>31</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Roboto</family>
       <pointsize>15</pointsize>
       <weight>50</weight>
       <italic>false</italic>
       <bold>false</bold>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">color: white;</string>
     </property>
     <property name="text">
      <string>nombre</string>
     </property>
    </widget>
    <widget class="QLabel" name="lblUsuario_5">
     <property name="geometry">
      <rect>
       <x>50</x>
       <y>120</y>
       <width>101</width>
       <height>31</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Roboto</family>
       <pointsize>15</pointsize>
       <weight>50</weight>
       <italic>false</italic>
       <bold>false</bold>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">color: white;</string>
     </property>
     <property name="text">
      <string>estado</string>
     </property>
    </widget>
    <widget class="QLabel" name="lblUsuario_6">
     <property name="geometry">
      <rect>
       <x>50</x>
       <y>190</y>
       <width>101</width>
       <height>31</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Roboto</family>
       <pointsize>15</pointsize>
       <weight>50</weight>
       <italic>false</italic>
       <bold>false</bold>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">color: white;</string>
     </property>
     <property name="text">
      <string>partido</string>
     </property>
    </widget>
    <widget class="QComboBox" name="inPartido">
     <property name="geometry">
      <rect>
       <x>50</x>
       <y>220</y>
       <width>271</width>
       <height>32</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">color: grey;
border: solid 0px white;
background: white;
border-radius:10px;
padding:8px;</string>
     </property>
    </widget>
    <widget class="QLabel" name="lblUsuario_7">
     <property name="geometry">
      <rect>
       <x>50</x>
       <y>260</y>
       <width>101</width>
       <height>31</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Roboto</family>
       <pointsize>15</pointsize>
       <weight>50</weight>
       <italic>false</italic>
       <bold>false</bold>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">color: white;</string>
     </property>
     <property name="text">
      <string>elecci√≥n</string>
     </property>
    </widget>
    <widget class="QComboBox" name="inEleccion">
     <property name="geometry">
      <rect>
       <x>50</x>
       <y>290</y>
       <width>271</width>
       <height>32</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">color: grey;
border: solid 0px white;
background: white;
border-radius:10px;
padding:8px;</string>
     </property>
    </widget>
    <widget class="QPlainTextEdit" name="inPropuesta">
     <property name="geometry">
      <rect>
       <x>370</x>
       <y>80</y>
       <width>321</width>
       <height>311</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>11</pointsize>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">color: grey;
border: solid 0px white;
background: white;
border-radius:10px;
padding:8px;</string>
     </property>
     <property name="plainText">
      <string/>
     </property>
    </widget>
    <widget class="QLabel" name="lblUsuario_8">
     <property name="geometry">
      <rect>
       <x>370</x>
       <y>50</y>
       <width>101</width>
       <height>31</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Roboto</family>
       <pointsize>15</pointsize>
       <weight>50</weight>
       <italic>false</italic>
       <bold>false</bold>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">color: white;</string>
     </property>
     <property name="text">
      <string>propuesta</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="inCurpCand">
     <property name="geometry">
      <rect>
       <x>50</x>
       <y>360</y>
       <width>271</width>
       <height>31</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>12</pointsize>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">color: grey;
border: solid 0px white;
background: white;
border-radius:10px;
padding:8px;</string>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QLabel" name="lblUsuario_17">
     <property name="geometry">
      <rect>
       <x>50</x>
       <y>330</y>
       <width>101</width>
       <height>31</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Roboto</family>
       <pointsize>15</pointsize>
       <weight>50</weight>
       <italic>false</italic>
       <bold>false</bold>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">color: white;</string>
     </property>
     <property name="text">
      <string>CURP</string>
     </property>
    </widget>
   </widget>
   <widget class="QWidget" name="vot">
    <widget class="QLabel" name="label_2">
     <property name="geometry">
      <rect>
       <x>210</x>
       <y>10</y>
       <width>291</width>
       <height>41</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Roboto</family>
       <pointsize>25</pointsize>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">color: white;</string>
     </property>
     <property name="text">
      <string>Registro de Votantes</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
    <widget class="QPushButton" name="btnVotante">
     <property name="geometry">
      <rect>
       <x>160</x>
       <y>410</y>
       <width>171</width>
       <height>31</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>-1</pointsize>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">border: 2px solid  #EF40A7;
background: rgba(0,0,0,0);
color:white;
border-radius: 15px;
font-size:15px;
</string>
     </property>
     <property name="text">
      <string>Soy Candidato</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="inNombre">
     <property name="geometry">
      <rect>
       <x>160</x>
       <y>100</y>
       <width>391</width>
       <height>31</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>12</pointsize>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">color: grey;
border: solid 0px white;
background: white;
border-radius:10px;
padding:8px;</string>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QLabel" name="lblUsuario">
     <property name="geometry">
      <rect>
       <x>160</x>
       <y>70</y>
       <width>101</width>
       <height>31</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Roboto</family>
       <pointsize>15</pointsize>
       <weight>50</weight>
       <italic>false</italic>
       <bold>false</bold>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">color: white;</string>
     </property>
     <property name="text">
      <string>nombre</string>
     </property>
    </widget>
    <widget class="QLabel" name="lblUsuario_2">
     <property name="geometry">
      <rect>
       <x>160</x>
       <y>140</y>
       <width>101</width>
       <height>31</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Roboto</family>
       <pointsize>15</pointsize>
       <weight>50</weight>
       <italic>false</italic>
       <bold>false</bold>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">color: white;</string>
     </property>
     <property name="text">
      <string>estado</string>
     </property>
    </widget>
    <widget class="QComboBox" name="inEstado">
     <property name="geometry">
      <rect>
       <x>160</x>
       <y>170</y>
       <width>391</width>
       <height>32</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">color: grey;
border: solid 0px white;
background: white;
border-radius:10px;
padding:8px;</string>
     </property>
    </widget>
    <widget class="QLabel" name="lblUsuario_3">
     <property name="geometry">
      <rect>
       <x>160</x>
       <y>210</y>
       <width>101</width>
       <height>31</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Roboto</family>
       <pointsize>15</pointsize>
       <weight>50</weight>
       <italic>false</italic>
       <bold>false</bold>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">color: white;</string>
     </property>
     <property name="text">
      <string>CURP</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="inCurp">
     <property name="geometry">
      <rect>
       <x>160</x>
       <y>240</y>
       <width>391</width>
       <height>31</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>12</pointsize>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">color: grey;
border: solid 0px white;
background: white;
border-radius:10px;
padding:8px;</string>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QPushButton" name="btnRegistrarVotante">
     <property name="geometry">
      <rect>
       <x>370</x>
       <y>410</y>
       <width>171</width>
       <height>31</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>-1</pointsize>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">border: 0px solid;
background: #EF40A7;
color:white;
border-radius: 15px;
font-weight:bold;
font-size:15px;
</string>
     </property>
     <property name="text">
      <string>REGISTRARSE</string>
     </property>
    </widget>
   </widget>
  </widget>
  <widget class="QPushButton" name="brnRegresar">
   <property name="geometry">
    <rect>
     <x>80</x>
     <y>40</y>
     <width>61</width>
     <height>51</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>-1</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">border: 0px solid;
background: #EF40A7;
color:white;
border-radius: 20px;
font-weight:bold;
font-size:25px;
</string>
   </property>
   <property name="text">
    <string>X</string>
   </property>
  </widget>
 </widget>
 <resources>
  <include location="resource.qrc"/>
 </resources>
 <connections/>
</ui>
